docs/developing/http-apis.md

```diff
@@ -169,6 +171,19 @@ automatically refreshes the IFRAME every 5 minutes.
   to redeem it as a sharing link of the form
   `https://sandstorm.example.com/shared/$API_TOKEN`.
 
+## Getting the user IP address
+
+By default, for privacy, Sandstorm removes the user's IP address from
+API requests. The API _client code_ can request that Sandstorm provide
+the IP address to the grain in an API request by setting a header:
+
+```
+X-Sandstorm-Passthrough: address
+```
+
+Your app can provide sample Javascript to make make API requests which
+sets this header on `XMLHttpRequest` calls to the API endpoint.
+
```

Comment:
It'd be great to mention that the IP will be available to your backend through the legacy-http-bridge via the `X-Real-IP` header.